### PR TITLE
Complete meeting

### PR DESCRIPTION
**Pull request by lalli-flores**
Plus control names match component default

### CODE REVIEW

File: src/MeetingsSDKAdapter.js [hunk 1]
```diff
@@ -131,11 +131,13 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
...
    *
-   * @param {string} ID  ID of the meeting to fetch
+   * @param {string} ID     ID of the meeting to fetch
+   * @param {Object} event  Event type that was triggered
+   * @param {Subject} end$  Subject used to complete @see {@link getMeeting}
    * @memberof MeetingsSDKAdapter
    * @private
    */
-  removeMedia(ID, {type}) {
+  removeMedia(ID, {type}, end$) {
```
Comment (akoushke) [inline src/MeetingsSDKAdapter.js]: no switch case needed here anymore. for now, `MEDIA:STOPPED` gets only triggered if we are leaving the meeting. Therefore, we should remove all attributes from the existing meeting object except the ID. to just keep record of it. NOTE: the ID is SDK MEETING ID which will be removed after leaving the meeting from the meetings collections

File: scripts/puppeteer.server.js [hunk 2]
```diff
@@ -19,13 +19,18 @@ function handleVideo() {
...
 function getMeeting() {
-  webexSDKAdapter.meetingsAdapter.getMeeting(MEETING_ID).subscribe((meeting) => {
-    document.getElementById('remote-audio').srcObject = meeting.remoteAudio;
-    document.getElementById('remote-video').srcObject = meeting.remoteVideo;
-    document.getElementById('local-audio').srcObject = meeting.localAudio;
-    document.getElementById('local-video').srcObject = meeting.localVideo;
-    document.getElementById('meeting-title').innerHTML = meeting.title;
-  });
+  webexSDKAdapter.meetingsAdapter.getMeeting(MEETING_ID).subscribe(
+    (meeting) => {
```
Comment (akoushke) [inline scripts/puppeteer.server.js]: can you log out the meeting object here as well?

File: src/MeetingsSDKAdapter.js [hunk 3]
```diff
@@ -7,8 +7,8 @@ const EVENT_MEDIA_STOPPED = 'media:stopped';
 const EVENT_MEDIA_LOCAL_UPDATE = 'adapter:media:local:update';
```
Comment (akoushke) [inline src/MeetingsSDKAdapter.js]: the test cases are not updated.

File: scripts/puppeteer.server.js [hunk 4]
```diff
@@ -83,7 +83,7 @@ document.getElementById('actions').addEventListener('click', async (event) => {
   try {
     switch (event.target.id) {
       case 'mute-audio':
-        await webexSDKAdapter.meetingsAdapter.meetingControls.audio.action(MEETING_ID);
+        await webexSDKAdapter.meetingsAdapter.meetingControls['mute-audio'].action(MEETING_ID);
```
Comment (akoushke) [inline scripts/puppeteer.server.js]: not part of your work but can you make the code look consistent? for both `audio` and `video` cases?

File: scripts/puppeteer.server.js [hunk 5]
```diff
@@ -83,7 +83,7 @@ document.getElementById('actions').addEventListener('click', async (event) => {
...
         break;
       case 'mute-video':
         await webexSDKAdapter.meetingsAdapter.handleLocalVideo(MEETING_ID);
```
Comment (akoushke) [inline scripts/puppeteer.server.js]: talking about this line and line 86

File: src/MeetingsSDKAdapter.js (hunk 1 above)
Comment (lalli-flores) [reply src/MeetingsSDKAdapter.js]: Removed switch case

**Pull request merged**